src/algebra/big_operators.lean

```diff
@@ -27,24 +27,57 @@ theorem finset.exists_le {α : Type u} [nonempty α] [directed_order α] (s : fi
 directed.finset_le (by apply_instance) directed_order.directed s
 
 namespace finset
-variables {s s₁ s₂ : finset α} {a : α} {f g : α → β}
 
-/-- `s.prod f` is the product of `f x` as `x` ranges over the elements of the finite set `s`. -/
-@[to_additive "`s.sum f` is the sum of `f x` as `x` ranges over the elements
+/-- `∏ x in s, f x` is the product of `f x` as `x` ranges over the elements of the finite set `s`. -/
+@[to_additive "`∑ x in s, f` is the sum of `f x` as `x` ranges over the elements
 of the finite set `s`."]
 protected def prod [comm_monoid β] (s : finset α) (f : α → β) : β := (s.1.map f).prod
 
+end finset
+
+/-
+## Operator precedence of `∏` and `∑`
+
+There is no established mathematical convention
+for the operator precedence of big operators like `∏` and `∑`.
+We will have to make a choice.
+
+Online discussions, such as https://math.stackexchange.com/q/185538/30839
+seem to suggest that `∏` and `∑` should have the same precedence,
+and that this should be somewhere between `*` and `+`.
+The latter have precedence levels `70` and `65` respectively,
+and we therefore choose the level `67`.
+
+In practice, this means that parentheses should be placed as follows:
+```lean
+∑ k in K, (a k + b k) = ∑ k in K, a k + ∑ k in K, b k →
+  ∏ k in K, a k * b k = (∏ k in K, a k) * (∏ k in K, b k)
+```
+(Example taken from page 490 of Knuth's *Concrete Mathematics*.)
+-/
+
+localized "notation `∑` binders `, ` r:(scoped:67 f, finset.sum finset.univ f) := r" in big_operators
```

Comment:
Could you mention the notations in the file docstring (talking both about the sum on a finset, and the sum on `finset.univ` in a fintype)?

Reply:
Done